src/pm-rpc/privates/appsRegistrar.js

```diff
@@ -13,4 +13,4 @@ export const hasApp = id => Boolean(_apps[id])
 
 export const unregisterApp = id => delete _apps[id]
 
-export const isEmpty = () => Object.keys(_apps).length > 0
\ No newline at end of file
+export const isEmpty = () => Object.keys(_apps).length === 0
```

Comment:
the check before is weird
but aren't you afaid it will break something ?

Reply:
if it does break, then it breaks a broken thing, I see no point in keeping broken things on code :)